doc/docs.md

```diff
@@ -608,6 +658,66 @@ age := 12
 println('age = $age')
 ```
 
+### Runes
+
+A `rune` represents a single Unicode character and is an alias for `u32`. To denote them, use `
+(backticks) :
+
+```v
+rocket := `🚀`
+```
+
+A `rune` can be converted to a UTF-8 string by using the `.str()` method.
+
+```v
+rocket := `🚀`
+assert rocket.str() == '🚀'
+```
+
+A `rune` can be converted to UTF-8 bytes by using the `.bytes()` method.
+
+```v failcompile
+rocket := `🚀`
+assert rocket.bytes() == [0xf0, 0x9f, 0x9a, 0x80]
```

Comment:
`assert rocket.bytes() == [byte(0xf0), 0x9f, 0x9a, 0x80]` compiles for me (the first cast is needed otherwise the right type is []int)

Reply:
Thanks.